cfgov/apache/conf.d/redirects.conf

```diff
@@ -37,13 +31,8 @@ RedirectMatch 404 \/yourstory\/.*\/[^$]
 # See discussion: https://GHE/eregs/eRegs-Notes/issues/2103#issuecomment-149183
 RedirectMatch  ^/eregulations/(.*)2015-26607_20180101(.*)$ /eregulations/$12017-18284_20180101$2
 
-# Ask CFPB
-Redirect permanent /askcfpb/askcfpb/js/ /static/askcfpb/js/
-Redirect permanent /askcfpb/askcfpb/img/ /static/askcfpb/img/
```

Comment:
It works fine, but it feels weird to have redirects for static asset paths living in the content management system now. No need to change anything here for now, but we might want to follow up to see why this redirect was needed and if there's a better way to do it now than a redirect.

Reply:
Agreed!